src/commands/kv/mod.rs

```diff
@@ -16,6 +17,19 @@ const INTERACTIVE_RESPONSE_LEN: usize = 1;
 const YES: &str = "y";
 const NO: &str = "n";
 
+
+// Get namespace id for a given binding name.
+pub fn get_namespace_id(target: &Target, title: &str) -> Option<String> {
+    if let Some(namespaces) = &target.kv_namespaces {
+        for namespace in namespaces {
+            if namespace.binding == title {
+                return Some(namespace.id.to_string());
```

Comment:
if we are not already, we should assert in the toml parsing that binding names are unique within a target.